sdks/python/apache_beam/utils/annotations.py

```diff
@@ -17,6 +17,17 @@
 
 """Deprecated and experimental annotations.
 
+Experimental: Signifies that a public API (public class, method or field) is
```

Comment:
I think this is important for contributors to know the distinction,
and I also believe we should make this information available to SDK users.

Could these paragraphs be also displayed somewhere on https://beam.apache.org,
(like we do with [snippets.py](https://github.com/apache/beam/blob/9d3ee79454a6065a71580730dbc2021f228266a2/sdks/python/apache_beam/examples/snippets/snippets.py#L314))?

Reply:
I agree that it would be good to expose this information on the web site, perhaps in the form of a page for api support. Filed: https://issues.apache.org/jira/browse/BEAM-4550